src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
We'd still need a specific way to pass the inverse tags through 2 layers of children though